saleor/graphql/salingo/utils.py

```diff
@@ -32,26 +33,30 @@ def get_receipt_payload(order):
     lines_json.append(shipping_position)
 
     summary = {
-        "to": price_to_minor_unit(value=order.total_gross_amount, currency='PLN')
+        "to": price_to_minor_unit(value=order.total_paid_amount, currency='PLN')
     }
 
     payload = {
         "lines": lines_json,
         "summary": summary,
     }
+    discounts = []
 
-    discount = get_voucher_discount_for_order(order)
-    if discount.amount > 0:
-        discount_position = [{
-            "type": "bill",
-            "discount": {
-                "na": order.voucher.code,
-                "rd": "true",
-                "rw": price_to_minor_unit(value=discount.amount, currency="PLN")
-            }
-        }]
-        payload["discounts"] = discount_position
+    order_manual_discounts = get_manual_discounts_for_order(order)
+    for order_discount in order_manual_discounts:
+        discount_name = "Discount" if order_discount.reason is None \
```

Comment:
order_discount.reason should be an empty string, so its never None. discount_name = order_discount.reason or "Discount" might be better